testsuite/MDAnalysisTests/utils/test_distances.py

```diff
@@ -325,7 +355,7 @@ def test_selfdist(self):
 
     def test_distarray(self):
         from MDAnalysis.lib.distances import distance_array
-        from MDAnalysis.lib.distances import transform_StoR, transform_RtoS
+        from MDAnalysis.lib.distances import transform_StoR
```

Comment:
They are most probably removed automatically by my IDE because they are not referenced anywhere. Should I put them back?